src/uu/split/src/split.rs

```diff
@@ -37,7 +37,7 @@ static OPT_NUMERIC_SUFFIXES_SHORT: &str = "-d";
 static OPT_HEX_SUFFIXES: &str = "hex-suffixes";
 static OPT_HEX_SUFFIXES_SHORT: &str = "-x";
 static OPT_SUFFIX_LENGTH: &str = "suffix-length";
-static OPT_DEFAULT_SUFFIX_LENGTH: &str = "0";
+static OPT_DEFAULT_SUFFIX_LENGTH: &str = "2";
```

Comment:
could you please add a comment explaining why "2" ?

Reply:
pushed an update with comment